tests/warp/warp_test.go

```diff
@@ -39,6 +41,12 @@ var (
fd5b600080fd5b6000819050919050565b61031981610306565b811461032457600080fd5b50565b60008135905061033681610310565b92915050565b600060208284031215610352576103516102fc565b5b600061036084828501610327565b91505092915050565b600080fd5b600080fd5b600080fd5b60008083601f84011261038e5761038d610369565b5b8235905067ffffffffffffffff8111156103ab576103aa61036e565b5b6020830191508360018202830111156103c7576103c6610373565b5b9250929050565b60008060008060008060a087890312156103eb576103ea6102fc565b5b60006103f989828a01610327565b965050602061040a89828a01610327565b955050604061041b89828a01610327565b945050606061042c89828a01610327565b935050608087013567ffffffffffffffff81111561044d5761044c610301565b5b61045989828a01610378565b92509250509295509295509295565b60008060008060608587031215610482576104816102fc565b5b600061049087828801610327565b94505060206104a187828801610327565b935050604085013567ffffffffffffffff8111156104c2576104c1610301565b5b6104ce87828801610378565b925092505092959194509250565b6000815190506104eb81610310565b92915050565b600060208284031215610507576105066102fc565b5b6000610515848285016104dc565b91505092915050565b600080fd5b6000601f19601f8301169050919050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b61056c82610523565b810181811067ffffffffffffffff8211171561058b5761058a610534565b5b80604052505050565b600061059e6102f2565b90506105aa8282610563565b919050565b600080fd5b600080fd5b600067ffffffffffffffff8211156105d4576105d3610534565b5b6105dd82610523565b9050602081019050919050565b60005b838110156106085780820151818401526020810190506105ed565b60008484015250505050565b6000610627610622846105b9565b610594565b905082815260208101848484011115610643576106426105b4565b5b61064e8482856105ea565b509392505050565b600082601f83011261066b5761066a610369565b5b815161067b848260208601610614565b91505092915050565b600060a0828403121561069a5761069961051e565b5b6106a460a0610594565b905060006106b4848285016104dc565b60008301525060206106c8848285016104dc565b60208301525060406106dc848285016104dc565b60408301525060606106f0848285016104dc565b606083015250608082015167ffffffffffffffff811115610714576107136105af565b5b61072084828501610656565b60808301525092915050565b60008115159050919050565b6107418161072c565b811461074c57600080fd5b50565b60008151905061075e81610738565b92915050565b6000806040838503121561077b5761077a6102fc565b5b600083015167ffffffffffffffff81111561079957610798610301565b5b6107a585828601610684565b92505060206107b68582860161074f565b9150509250929050565b600081905092915050565b82818337600083830152505050565b60006107e683856107c0565b93506107f38385846107cb565b82840190509392505050565b600061080c8284866107da565b91508190509392505050565b61082181610306565b82525050565b600082825260208201905092915050565b60006108448385610827565b93506108518385846107cb565b61085a83610523565b840190509392505050565b600060608201905061087a6000830187610818565b6108876020830186610818565b818103604083015261089a818486610838565b90509594505050505056fea2646970667358221220b2cefd832fa6ca66aff8dba2e446584042974aa24a3f54e0f99e83fc9cf34edf64736f6c63430008140033")
```

Comment:
Is there a better way to do this rather than having a massive line like this?

Reply:
Previously, I used a goembed function, keeping the abi and bin in a separate folder.  Aaron suggested it might be better to keep the data directly in this file.  In other files contract abi/bins have been stored directly like this, so I think we're mostly trying to adhere to the same style (though those have mostly been much shorter)

Reply:
I think we should embed this.

Reply:
I think it makes everything a little cleaner, I added it back in